hnn_core/params.py

```diff
@@ -190,48 +190,49 @@ def _feed_validate(p_ext, p_ext_d, tstop):
     ----------
     p_ext : list
         Cumulative list of dicts where each dict contains
-        all parameters of an extinput.
+        all parameters of an ExtFeed input.
     p_ext_d : dict
-        The extinput to validate and append to p_ext.
+        The parameter set to validate and append to p_ext.
     tstop : float
         Stop time of the simulation.
 
     Returns
     -------
     p_ext : list
-        Cumulative list of dicts with newly appended
-        extinput.
+        Cumulative list of dicts with newly appended ExtFeed.
     """
 
-    # # reset tstop if the specified tstop exceeds the
-    # # simulation runtime
-    # if p_ext_d['tstop'] == 0:
-    #     p_ext_d['tstop'] = tstop
-
-    if p_ext_d['tstop'] > tstop:
-        p_ext_d['tstop'] = tstop
-
-    # if stdev is zero, increase synaptic weights 5 fold to make
-    # single input equivalent to 5 simultaneous input to prevent spiking
-    # <<---- SN: WHAT IS THIS RULE!?!?!?
-    if not p_ext_d['stdev'] and p_ext_d['distribution'] != 'uniform':
-        for key in p_ext_d.keys():
-            if key.endswith('Pyr'):
-                p_ext_d[key] = (p_ext_d[key][0] * 5., p_ext_d[key][1])
-            elif key.endswith('Basket'):
-                p_ext_d[key] = (p_ext_d[key][0] * 5., p_ext_d[key][1])
-
-    # if L5 delay is -1, use same delays as L2 unless L2 delay is 0.1 in
-    # which case use 1. <<---- SN: WHAT IS THIS RULE!?!?!?
-    if p_ext_d['L5Pyr_ampa'][1] == -1:
-        for key in p_ext_d.keys():
-            if key.startswith('L5'):
-                if p_ext_d['L2Pyr'][1] != 0.1:
-                    p_ext_d[key] = (p_ext_d[key][0], p_ext_d['L2Pyr'][1])
-                else:
-                    p_ext_d[key] = (p_ext_d[key][0], 1.)
-
-    p_ext.append(p_ext_d)
+    # only append if t0 is less than simulation tstop
+    if tstop > p_ext_d['t0']:
```

Comment:
is this a rebase issue or was this undone in your previous PR somehow? I seem to recall seeing this diff in @rythorpe 's PR

Reply:
This was undone from my previous PR and is evident in the change I reviewed above in `test_network.py`. Even though this appears to fix the discrepancy with the alpha simulation, I don't think this is a good solution because it results in no `common` input gids being created in the network object.